MekHQ/src/mekhq/campaign/market/contractMarket/AtbMonthlyContractMarket.java

```diff
@@ -473,7 +472,7 @@ public double calculatePaymentMultiplier(Campaign campaign, AtBContract contract
             }
         }
 
-        multiplier *= contract.getContractType().getPaymentMultiplier();
+        multiplier *= contract.getContractType().getOperationsTempoMultiplier();
```

Comment:
Thanks for including this. Will make implementing Fame & Infamy modifiers _much_ easier.